gdal/swig/perl/t/00.t

```diff
@@ -62,7 +62,14 @@ for my $type (Geo::GDAL::DataTypes()) {
 # sub DecToPackedDMS ok
 
 my $dms = Geo::GDAL::DecToDMS(62, 'Long');
-ok($dms eq " 62d 0' 0.00\"E", "DecToDMS, got '$dms'"),
+ok($dms =~ " 62[°d] 0' 0.00\"E", "DecToDMS, got '$dms'"),
```

Comment:
As Perl bindings will be removed in GDAL 3.5, hence those tests too, would you mind writing them in the Python autotest suite instead or in autotest/cpp ?

Reply:
I'll have a go.
Python autotests are failing on my machine at
    from osgeo import gdal

Reply:
maybe check that you haven't already "system" python bindings that interfer with. Try  "source scripts/setdevenv.sh" from gdal/

Reply:
If you still have issues with Python then provide more details and I'll try to help.

Reply:
Thanks. I've made progress.
This is Ubuntu 21.04: the python 3 pytest is called pytest-3, so I was running a copy of the python2 version. Do we need to get configure to look for "pytest-3" ?

Currently pytest-3 is aborting with 
gcore/misc.py::test_misc_5 terminate called after throwing an instance of 'std::length_error'
  what():  basic_string::_M_create
Fatal Python error: Aborted

Full pytest-3 -vvs  output 
[pytest.errlog.20210503.txt](https://github.com/OSGeo/gdal/files/6415044/pytest.errlog.20210503.txt)

But I'm having a day off and not stuff yet.

Reply:
not sure why you this error. you might try a "make clean" and rebuild : GDAL build scripts don't track header file dependencies

Reply:
Yes, it was something like that.